.github/workflows/test.yml

```diff
@@ -64,6 +64,7 @@ jobs:
         uses: Ana06/get-changed-files@v2.2.0 # it's a fork of jitterbit/get-changed-files@v1 which works better with pull requests
         with:
           format: 'json'
+        continue-on-error: true
       - id: detect
         run: ./.github/workflows/scripts/detect-jobs-to-run.js <<<'${{ steps.files.outputs.all }}'
```

Comment:
Let's try this, though 100% confident this will work so let's watch how it goes

It relies on the fallback to "all" on https://github.com/prisma/prisma/blob/main/.github/workflows/scripts/detect-jobs-to-run.js

Reply:
Oh yes, I should have added that as a comment. Also not sure how this will exactly be displayed etc :/ 

Pierre has a solution that is most probably cleaner, but requires a lot more diff: https://github.com/prisma/prisma/pull/17598